### DISCUSSION TITLE
Is there a way to batch load .trc files in sequence, graph definition then save the graph image.

### DISCUSSION THREAD

**Discussion by ArcticBeat05** [collin80/SavvyCAN#483] [Q&A]
I notice there is a scripting interface, can this be done in there? 
Load .trc file AND graph definition file -> Save Graph image automatically.

**Comment by collin80** [comment]
Currently there isn't any way to do that. The scripting interface is basically only useful to do custom CAN bus stuff with sending and receiving messages. I'm investigating having python scripting available and it may be possible to then use python to bring together the various systems within SavvyCAN to automatically do these things.
